docs/reporters.md

```diff
@@ -6,7 +6,7 @@ title: InSpec Reporters
 
 Introduced in InSpec 1.51.6
 
-InSpec allows you to output your test results to one or more reporters. You can configure the reporter(s) using either the `--json-config` option or the `--reporter` option. While you can configure multiple reporters to write to different files, only one reporter can output to the screen(stdout).
+InSpec allows you to output your test results to one or more reporters. You can configure the reporter(s) using either the `--config` (or `--json-config`, prior to v3.6) option or the `--reporter` option. While you can configure multiple reporters to write to different files, only one reporter can output to the screen(stdout).
```

Comment:
```suggestion
InSpec allows you to output your test results to one or more reporters. Configure the reporter(s) using either the `--config` (or `--json-config`, prior to v3.6) option or the `--reporter` option. While you can configure multiple reporters to write to different files, only one reporter can output to the screen(stdout).
```

Also, it would be great to define what a reporter is at the top of this doc.